an-der-uni/ausland.tex

```diff
@@ -9,5 +9,7 @@
 
 AuslandsBAföG müsst ihr übrigens separat beantragen, die Zu\-stän\-dig\-keit dafür hängt vom jeweiligen Ausland ab, eine Liste der zuständigen Ämter findet man z.B. unter \url{AuslandsBAföG.de}. In Heidelberg kann man sich auch direkt an das Dezernat Internationale Beziehungen (früher Akademisches Auslandsamt, AAA)\footnote{\url{http://www.uni-heidelberg.de/studium/kontakt/auslandsamt/}} wenden, das ihr in der Seminarstraße 2 findet. Ihr müsst dann ins Zimmer 139.
 
+Sehr wichtig ist noch zu wissen, dass man Urlaubssemester beantragen kann, damit die Fachsemesterzahl (vorallem wichtig für alle, die Inlands-BAföG erhalten) nicht weiterläuft. Dafür läuft die Hochschulsemesterzahl weiter. Man kann sich dann die im Ausland erworbenen CPs (nach Absprache) dann anrechnen lassen.
```

Comment:
Wo kann man Urlaubssemester beantragen und mit wem muss die "Absprache" stattfinden?
Wäre gut, wenn diese Info noch in den Text kommt.
Außerdem sollte überall einheitlich **LP** statt CP stehen.